src/presentation/http/router/auth.test.ts

```diff
@@ -0,0 +1,67 @@
+import { describe, test, expect } from 'vitest';
+
+describe('Auth API', () => {
+  describe('POST /auth', () => {
+    test('Returns 401 when session is not valid', async () => {
+      const expectedStatus = 401;
+      const refreshToken = 'not-validToken';
+      const response = await global.api?.fakeRequest({
+        method: 'POST',
+        url: '/auth',
+        body: { token: refreshToken },
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+
+      const body = response?.body !== undefined ? JSON.parse(response?.body) : {};
+
+      expect(body).toStrictEqual({ message: 'Session is not valid' });
+    });
+
+    test('Returns 200 when session was authorized', async () => {
+      const expectedStatus = 200;
+
+      /** Define the token to include in the request body */
+      const refreshToken = 'pv-jIqfPj1';
```

Comment:
this refresh token belongs to session with past date. If you try to re-run this test, it will fail. You need to dynamically set future expiration date when appending data to test DB in order to make the test always work